aws-lambda/README.md

```diff
@@ -1,3 +1,3 @@
 # AWS Lambda Application Template
 
-This is a base template for running AWS Lambda Functions on Massdriver provisioned infrastructure.
\ No newline at end of file
+This is a base template for running AWS Lambda Functions on Massdriver provisioned infrastructure. Examples of adding triggers to the base template can be seen [here](https://github.com/massdriver-cloud/application-examples/aws-lambda)
```

Comment:
Do we have an example app? If so let’s nest it under AWS/ like k8s/ So we don’t have 1000 examples have the top level. We’re trying to move the other repos with a lot of top level directories into subdirectories

Reply:
I am doing that with the example app, dockerfile and instructions in the application examples repo. I will nest under AWS.